app/models/katello/candlepin/product_content.rb

```diff
@@ -34,5 +34,15 @@ def content_override(activation_key)
       override = activation_key.content_overrides.find { |pc| pc[:contentLabel] == content.label }
       override.nil? ? 'default' : override[:value]
     end
+
+    def content_type
+      self.content.type
+    end
+
+    def displayable?
+      return true  unless (content_type == ::Katello::Repository::CANDLEPIN_DOCKER_TYPE) ||
+                          (::Katello::RepositoryTypeManager.find(Repository::OSTREE_TYPE).nil? &&
+                            content_type == ::Katello::Repository::CANDLEPIN_OSTREE_TYPE)
```

Comment:
This could just return the value of this expression since it results in a boolean.